code/modules/reagents/Chemistry-Recipes/Others.dm

```diff
@@ -351,4 +351,18 @@
 	id = "saltpetre"
 	result = "saltpetre"
 	required_reagents = list("potassium" = 1, "nitrogen" = 1, "oxygen" = 3)
-	result_amount = 3
\ No newline at end of file
+	result_amount = 3
+
+/datum/chemical_reaction/phosphorus_trichloride
+	name = "Phosphorus Trichloride"
+	id = "phosphorus_trichloride"
+	result = "phosphorus_trichloride"
+	required_reagents = list("phosphorus" = 1, "chlorine" = 3)
+	result_amount = 4
+
+/datum/chemical_reaction/methyl_phosphonous_dichloride
```

Comment:
This recipe and the one above seem to refer to a chemical not in the code in any way